extensions/DynamicPageList/DPLSetup.php

```diff
@@ -1391,7 +1391,7 @@ public static function dplParserFunction(&$parser) {
 
     }
 
-    public static function dplNumParserFunction(&$parser, $text='') {
+    public static function dplNumParserFunction($parser, $text='') {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)